src/unstract/sdk/vector_db.py

```diff
@@ -76,10 +76,18 @@ def get_vector_db(
                     )
                     return vector_db_adapter_class.get_vector_db_instance()
                 else:
-                    return None
+                    raise SdkError(
```

Comment:
@gaya3-zipstack please update the return type to remove `None` here `Union[BasePydanticVectorStore, VectorStore, None]`

Reply:
Done. Thank you... !!